src/go/rpk/pkg/cli/registry/schema/get.go

```diff
@@ -123,7 +137,31 @@ potential (mutually exclusive) ways:
 	cmd.Flags().StringVar(&schemaFile, "schema", "", "Schema file to check existence of, must be .avro or .proto; subject required")
 	cmd.Flags().StringVar(&schemaType, "type", "", fmt.Sprintf("Schema type of the file used to lookup (%v); overrides schema file extension", strings.Join(supportedTypes, ",")))
 	cmd.Flags().BoolVar(&deleted, "deleted", false, "If true, also return deleted schemas")
+	cmd.Flags().BoolVar(&printSchema, "print-schema", false, "If true, print the schema JSON")
 
 	cmd.RegisterFlagCompletionFunc("type", validTypes())
 	return cmd
 }
+
+func printSchemaString(ss []sr.SubjectSchema) {
+	if len(ss) == 0 {
+		fmt.Println("No schema was found")
+	} else {
+		// This command finds either a specific schema or all subjects using
+		// the same schema. The first element in 'ss' always provides the
+		// desired schema.
+		s := ss[0].Schema
+		if s.Type == sr.TypeProtobuf {
+			fmt.Println(s.Schema)
```

Comment:
Should this have a return here?

Reply:
Thanks! good catch, I missed that entirely and only added the return a few lines (L64) below.